README.md

```diff
@@ -15,18 +15,364 @@ Install using composer:
 composer require utopia-php/orchestration
 ```
 
-Init in your application:
+### Example
 ```php
 <?php
 
-require_once __DIR__ . '/../../vendor/autoload.php';
+require_once 'vendor/autoload.php';
 
 use Utopia\Orchestration\Orchestration;
```

Comment:
![Screenshot 2022-03-25 at 3 40 14 PM](https://user-images.githubusercontent.com/20852629/160114467-62960f24-4054-441f-a2ce-0c02f40cccca.png)

Not an issue with your PR but I noticed a typo. So please fix it as well

Reply:
Fixed 👍